spec/app/models/mdm/host_spec.rb

```diff
@@ -547,14 +547,14 @@ def search_for(str)
         it "when the string contains 'ppc'" do
           expect(host.send(:get_arch_from_string, 'blahppcblah')).to eq('PowerPC')
         end
-      end
 
-      context 'should return nil' do
         it 'when PowerPC is cased incorrectly' do
-          expect(host.send(:get_arch_from_string, 'powerPC')).to eq(nil)
-          expect(host.send(:get_arch_from_string, 'Powerpc')).to eq(nil)
+          expect(host.send(:get_arch_from_string, 'powerPC')).to eq('PowerPC')
```

Comment:
Looks like this behavior changed with this PR to recog https://github.com/rapid7/recog/pull/406/files#diff-b8ba9b5ec82acf9a7399fd5884b901765ead4be11c4e7a21575db2aa664a2866R19

```
$ bundle exec ruby -e "require 'recog'; puts Recog::VERSION; puts Recog::Nizer.match('architecture', 'powerPC').inspect"
2.3.22
nil

$ bundle exec ruby -e "require 'recog'; puts Recog::VERSION; puts Recog::Nizer.match('architecture', 'powerPC').inspect"
2.3.23
{"matched"=>"PowerPC", "os.arch"=>"PowerPC", "service.protocol"=>"", "fingerprint_db"=>"architecture"}
```

Sync'ing up with @mkienow-r7 it seems good to update this test's behavior to align here, as the PR was making PowerPC to be consistent with the rest of the other `xml/architecture.xml` matches